src/index.ts

```diff
@@ -178,6 +178,7 @@ export * as redis from './infra/redis/index.js';
 export * as scheduler from './infra/scheduler/index.js';
 export * as cache from './infra/cache/index.js';
 export * as config from './lib/config.js';
+export * as registry from './lib/registry.js';
```

Comment:
Exporting the utility to make delete calls to the registry so we can reuse it to delete deltas from the registry in balena-api